### PR TITLE
Create ``Textarea`` component

### PR DESCRIPTION
**Pull request by PavelHolec**
Unified coordinator to work for both InputField and Textarea, so that the event API is consistent no matter what mix of these two text input components call site uses.

https://github.com/kiwicom/orbit-swiftui/assets/35844477/fc8dcda5-bb83-4a66-abf7-652342354879

### CODE REVIEW

File: Sources/Orbit/Support/TextFields/TextView.swift [hunk 1]
```diff
@@ -0,0 +1,283 @@
+import SwiftUI
+import UIKit
+
+/// Orbit control that displays a multiline editable text interface, a replacement for the native multiline `TextField` component.
```
Comment (sjavora) [inline Sources/Orbit/Support/TextFields/TextView.swift:4]: So what's the difference between this and `Textarea`? Why do we have more than one multiline text input?

File: Sources/Orbit/Support/TextFields/TextView.swift [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+///
+/// The component uses UIKit `UITextView` implementation to support these feature for older iOS versions:
+/// - focus changes
+/// - UITextField event handling
+/// - full UITextField configuration
+/// - font and text override
+/// - larger and configurable touch area
+///
+/// The component is compatible with native `@FocusState` modifier to support focus changes in later iOS versions.
```
Comment (sjavora) [inline Sources/Orbit/Support/TextFields/TextView.swift:13]: newer

File: Sources/Orbit/Components/Textarea.swift [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+/// ```
+/// 
+/// The component uses a custom ``TextView`` component (implemented using `UITextView`) that represents the native `TextField`.
+///
+/// ### Layout
+///
+/// The component expands horizontally and vertically, unless limited by a `frame` modifier.
+/// 
+/// - Note: [Orbit definition](https://orbit.kiwi/components/input/textarea/)
+public struct Textarea: View, TextFieldBuildable {
```
[the remark this answers is not in the document]
Comment (PavelHolec) [reply Sources/Orbit/Components/Textarea.swift:24]: The naming is taken from [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea), so it is intentional from Orbit side. As with other components, we should follow with the same name, as-is, even if it sounds weird.

File: Sources/Orbit/Components/Textarea.swift [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            messageHeight: $messageHeight
+        ) {
+            InputContent(
+                state: state,
+                message: message,
+                isFocused: isFocused
+            ) {
+                textView
+            }
+        }
```
[the remark this answers is not in the document]
Comment (PavelHolec) [reply Sources/Orbit/Components/Textarea.swift]: Updated in other files as well

File: Sources/Orbit/Support/TextFields/InsetableTextView.swift [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        
+        addSubview(promptLabel)
+    }
+    
+    public override func layoutSubviews() {
+        super.layoutSubviews()
+        contentInset = insets
+//        promptLabel.preferredMaxLayoutWidth = bounds.width - (insets.left + insets.right)
+        let size = promptLabel.sizeThatFits(bounds.inset(by: insets).size)
+//        promptLabel.frame = .init(origin: .init(x: insets.left, y: insets.top), size: size)
```
[the remark this answers is not in the document]
Comment (PavelHolec) [reply Sources/Orbit/Support/TextFields/InsetableTextView.swift]: Updated

File: Sources/Orbit/Support/TextFields/InsetableTextView.swift [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+        }
+
+        return success
+    }
+
+    func updatePromptVisibility() {
+        promptLabel.isHidden = text.isEmpty == false
+    }
+    
+    public func replace(withText text: String, keepingCapacity: Bool = false) {
```
[the remark this answers is not in the document]
Comment (PavelHolec) [reply Sources/Orbit/Support/TextFields/InsetableTextView.swift]: Updated

File: Sources/Orbit/Support/TextFields/TextView.swift (hunk 2 above)
Comment (PavelHolec) [reply Sources/Orbit/Support/TextFields/TextView.swift]: Updated

**Review by sjavora — APPROVED** [review]

**Pull request merged**